crates/noirc_frontend/src/ast/expression.rs

```diff
@@ -636,12 +636,20 @@ impl Display for FunctionDefinition {
             format!("{name}: {visibility} {type}")
         });
 
+        let where_clause = vecmap(&self.where_clause, ToString::to_string);
+        let where_clause_str = if where_clause.len() > 0 {
```

Comment:
Clippy lint here to use `is_empty` instead

Reply:
Done.